addons/website_forum/static/src/js/website_forum.js

```diff
@@ -183,6 +185,19 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                     .fromSQL(post.dataset.lastActivity, {zone: 'utc'})
                     .toRelative();
             });
+
+        // show the sign-up call to action if appropriate
+        const signUpCta = document.querySelector(".o_wforum_sign_up_cta");
+        if (signUpCta) {
+            // check if public user has hidden the cta forever or temporarily (~one day)
+            const hiddenAt = parseInt(localStorage.getItem("forum-signup-cta-hidden-at")) || 0;
```

Comment:
Usual naming for those is more "website_forum.signup_cta_hidden"

So module_name.key_name_underscored. 

We could also stored "forever" inside this key OR a date, that way you only need one key.
You test if it's there, if it's there you first test if it's equal to "forever" and if not you test against your date delta.